src/pages/Landing/Upgrades/index.tsx

```diff
@@ -85,43 +85,43 @@ export const Phases = (): JSX.Element => {
           <div className="px20">
             <ScrollAnimation animateIn="fadeIn" animateOnce>
               <StyledHeading margin="none" className="mt20" isMobile={m}>
-                What is eth2 phase 0?
+                How does eth2 upgrade Ethereum?
               </StyledHeading>
             </ScrollAnimation>
             <ScrollAnimation animateIn="fadeInUp" animateOnce>
               <Text className="mt20">
-                Transitioning from proof-of-work (PoW) to proof-of-stake (PoS),
-                eth2 will be launched in at least 4 phases. The idea is to break
-                up this transition into phases each focusing on a different
-                aspect of eth2.
-              </Text>
-              <Text className="mt20">
-                <PhaseTitle>Phase 0 </PhaseTitle>
-                contains all of the machinery behind eth2&apos;s PoS consensus,
-                it tracks the validators and their balances.
+                In order to future-proof Ethereum, eth2 will upgrade various
+                components of Ethereum while seamlessly continuing on the chain
+                of today. The main components of this upgrade are:
               </Text>
               <Text className="mt20">
-                <PhaseTitle>Phase 1 </PhaseTitle>
-                handles adding, storing, and retrieving the data associated with
-                eth2's shards.
+                <UpgradeTitle>Proof of Stake </UpgradeTitle>
+                (PoS) is more secure, decentralised, and environmentally
+                friendly consensus mechanism when compared with Proof of Work
+                (PoW) that secures Ethereum today. Proof of Stake works by
+                having validators lock up {PRICE_PER_VALIDATOR} {TICKER_NAME}{' '}
+                and rewarding them for building the chain and punishing them if
+                they try to attack it. This portion of eth2 is already live in
+                the form of the beacon chain.
               </Text>
               <Text className="mt20">
-                <PhaseTitle>Phase 1.5 </PhaseTitle>
-                updates Ethereum as we know it today from PoW to PoS by making
-                it a shard under eth2.
+                <UpgradeTitle>The Merge </UpgradeTitle>
```

Comment:
I did see this on ethereum.org. I prefer The Merge because:
* I am more familiar with it
* It doesn't imply things can be un-docked
* The result sounds like one cohesive thing instead of two things duct-taped together

However, I am happy having it changed for consistency.